nilearn/image/image.py

```diff
@@ -798,6 +808,16 @@ def threshold_img(img, threshold, mask_img=None, copy=True):
         voxels which have intensities greater than this score will be kept.
         The given string should be within the range of "0%" to "100%".
 
+    cluster_threshold : :obj:`float`, optional
+        Cluster size threshold, in voxels. In the returned thresholded map,
+        sets of connected voxels (``clusters``) with size smaller
+        than this number will be removed. Default=0.
+
```

Comment:
Can we add a versionadded directive for these two new arguments ?

Reply:
Done!